### PR TITLE
Fix bug in paratii eth vids

### PR DESCRIPTION
**Pull request by bent0b0x**
image

This PR fixes this error when calling `paratii.eth.vids.get()`.

### CODE REVIEW

File: lib/paratii.eth.vids.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class ParatiiEthVids {
   async get (videoId) {
     let contract = await this.getRegistry()
     // await contract.setProvider(this.eth.config.provider)
-    let videoInfo = await contract.methods.get(videoId).call()
+    let videoInfo = await contract.methods.getVideoInfo(videoId).call()
```
Comment (bent0b0x) [inline lib/paratii.eth.vids.js]: This is the important change

**Comment by bent0b0x** [comment]
Well now the tests are failing saying `getVideoInfo` does not exist, so would definitely appreciate some guidance as to why this bug exists :)

**Comment by bent0b0x** [comment]
Hitting similar issues running `chimp` tests locally. What is a reason certain contracts may not be found even if `deployContracts` has been called?

**Comment by jellegerbrandy** [comment]
@bent0b0x , these issues are because, for some reason, you have an old version of `paratii-contracts`. 
For now, the (my) policy is to add explit git commits in pacage.json (    "paratii-contracts": "github:Paratii-Video/paratii-contracts#fc5949dcfe166ebeab702609d9f3035fd361416d") which seems to be the most robust way keep the exact right version. 

So the problem should be resolved with a fresh `git pull; yarn isntall`

I'm closing the PR.

**Pull request closed without merging**